### PR TITLE
Niad 324 aws document db tls

### PR DESCRIPTION
**Pull request by jaff47**

### CODE REVIEW

File: terraform/aws/components/base/variables.tf [hunk 1]
```diff
@@ -88,7 +88,7 @@ variable "docdb_master_password" {
 
 variable "docdb_tls" {
   type = string
-  default = "disabled"
+  default = "enabled"
```
Comment (LukaszKwasniewski) [inline terraform/aws/components/base/variables.tf:91]: make this enabled / disabled on per environment basis - move it to etc/

Comment (LukaszKwasniewski) [inline terraform/aws/components/base/variables.tf]: I guess better option would be to set the default to disabled (like it already is) and enable it via variable set in etc

File: terraform/aws/components/nhais/variables.tf [hunk 2]
```diff
@@ -78,7 +78,19 @@ variable "mq_broker_name" {
 variable "nhais_mongo_options" {
   type = string
   description = "Options for Mongo"
-  default = "replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false"
+  default = "replicaSet=rs0&readPreference=secondaryPreferred&retryWrites=false&tls=true"
```
Comment (LukaszKwasniewski) [inline terraform/aws/components/nhais/variables.tf]: As TF may not allow default for variable to be based on other variable we may need to remove the default here and set it in global without the tls and with tls on envs where it is enabled

File: terraform/aws/components/nhais/variables.tf [hunk 3]
```diff
@@ -78,7 +78,19 @@ variable "mq_broker_name" {
...
+}
+
+variable "nhais_mongo_documentdb_tls_enabled" {
+  type = string
+  description = "Enable TLS on AWS Document DB"
+  default = "true"
```
Comment (LukaszKwasniewski) [inline terraform/aws/components/nhais/variables.tf]: same as with base, I guess default is false, the actual value set in etc/ files.

Comment (grzegorz-robak) [reply terraform/aws/components/nhais/variables.tf]: default is false, we set true only for exemplar

Comment (jaff47) [reply terraform/aws/components/nhais/variables.tf]: It should be only enabled on V&P now, once testing is completed will apply to PTL (exemplar)

**Review by LukaszKwasniewski — APPROVED** [review]

**Pull request merged**